engine/security/src/main/scala/pl/touk/nussknacker/ui/security/basicauth/BasicAuthenticationConfiguration.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.ui.security.basicauth
+
+import com.typesafe.config.Config
+import pl.touk.nussknacker.ui.security.api.{AuthenticationConfiguration, CachingHashesConfig}
+
+import java.net.URI
+
+case class BasicAuthenticationConfiguration(usersFile: URI,
+                                            cachingHashes: Option[CachingHashesConfig]) extends AuthenticationConfiguration {
+
+  override def method: String = BasicAuthenticationConfiguration.name
+
+  def cachingHashesOrDefault: CachingHashesConfig = cachingHashes.getOrElse(CachingHashesConfig.defaultConfig)
```

Comment:
I know it's not your change, but this can be misleading - defaultConfig should be "merged" with the one configured by user...

Reply:
I do not fully understand what you mean by "merged" and do not want to handle it now since the PR has a different purpose. I just moved the `CachingHashaesConfig` class to the `basicauth` package where the class is solely used.

Reply:
ok, let's leave it for other time ;)